### ISSUE TITLE
load_state_dict

### ISSUE THREAD

**Issue by YiDanCheung** [sicara/easy-few-shot-learning#157]
Thank you for your outstanding work!

But I'm wondering how to use `load_state_dict()`, in [episodic_training.ipynb](https://github.com/sicara/easy-few-shot-learning/blob/master/notebooks/episodic_training.ipynb) you use `few_shot_classifier.load_state_dict(best_state)`, but [classical_training.ipynb](https://github.com/sicara/easy-few-shot-learning/blob/master/notebooks/classical_training.ipynb) you use `model.load_state_dict(best_state)` while the the former is PrototypicalNetworks and the latter is resnet12.

Looking for your reply!

**Comment by ebennequin** [comment]
It's because in episodic training we train the few-shot classifier, while in classical training we only train the feature extractor.

**Comment by YiDanCheung** [comment]
> It's because in episodic training we train the few-shot classifier, while in classical training we only train the feature extractor.

So maybe in [classical_training.ipynb](https://github.com/sicara/easy-few-shot-learning/blob/master/notebooks/classical_training.ipynb) `best_state = copy.deepcopy(few_shot_classifier.state_dict())` should be `best_state = copy.deepcopy(model.state_dict())`, and it would not cause error like `RuntimeError: Error(s) in loading state_dict for ResNet : Missing key(s) in state_dict: "conv1.weight", ...`

**Comment by ebennequin** [comment]
Indeed! It's a mistake that we introduced in [this commit](https://github.com/sicara/easy-few-shot-learning/commit/2ed387ef1e30176c679657ccca1b1a2b0ca247a4#diff-af5edba59ec2c961a08278ed1d220c4e4edf6b600ef82998723188a2b992720d). We should make the change.
